roles/ceph-common/templates/restart_osd_daemon.sh.j2

```diff
@@ -13,7 +13,11 @@ check_pgs() {
     let RETRIES=RETRIES-1
   done
   # PGs not clean, exiting with return code 1
-  echo "Error with PGs, check config"
+  echo "Error while running `$CEPH_CLI -s`, PGs were not reported as active+clean"
```

Comment:
actually, it should be `ceph $CEPH_CLI -s`. You are right that we show the status after, but we don't say what exactly we were running which I think it is important.